internal/services/admin/contracts/workflows.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.28.1
-// 	protoc        v5.29.3
+// 	protoc        v5.29.2
```

Comment:
Let's ignore these changes from this PR?

Reply:
good call, this keeps happening to me every time I run `task-setup` 🤷